### DISCUSSION TITLE
How to Create a Shared Folder on my Boot Drive (NVME) and use it?

### DISCUSSION THREAD

**Discussion by davedoc1984** [retronas/discussions#116] [Q&A]
<img width="556" alt="image" src="https://github.com/retronas/retronas/assets/97349996/ecf1cd8e-d1fa-4933-ae8f-8f35d9d514d1">

How can I create a Shared Folder on my NVME SSD (which is my Boot Drive for RetroNAS) so I can make use of the 400+GB's of Storage that has on it? I have done various googling around but can't for the life of me figure out out to create a folder on that Device and make it so it works in the same way as the Shares do for MiSTer, PS2, PS3 etc etc.

**Comment by sairuk** [comment]
This is a generic linux operation, nothing to do with RetroNAS specifically and we don't support custom shares. Although you can run custom playbooks these days.

You haven't mentioned the kind of share so I'll assume samba, and i'm not going to pretend to know how to do it through the cockpit interface off the top of my head, so he is the basic cli process.

Samba (as root)
1. create a directory (`mkdir /shared`)
2. set permissions (`chown <retronasuser>: /shared`)
3. install samba through retronas
4. change to the samba dir (`cd /etc/samba`)
5. cp `retronas.conf` to a new file (`cp retronas.conf davedoc.conf)
6. edit `davedoc.conf` and change the comment and path declaration
7. restart samba (`systemctl restart smbd`) or reboot

example config
```
comment = davedoc-share
path = /shared
guest ok = no
browseable = yes
write list = retronas
writeable = yes
force user = retronas
force group = retronas
create mask = 0775
directory mask = 0775
```

I should add I don't recommend you do this without additional controls like quotas or partitioning because you'll no doubt get to a point where you'll exhaust your root disk storage and either it wont boot or your break the os during an update
